tests/test_cnes_sonarqube.py

```diff
@@ -0,0 +1,251 @@
+"""
```

Comment:
Codacy found an issue: [Multi-line docstring summary should start at the first line (D212)](https://app.codacy.com/gh/cnescatlab/sonarqube/pullRequest?prid=6252252)